index.bs

```diff
@@ -59,6 +59,10 @@ urlPrefix: https://url.spec.whatwg.org/; spec: URL
     url: #percent-encode; type: abstract-op; text: percent-encode
 urlPrefix: https://www.w3.org/TR/payment-request/; spec: payment-request
     url: #dfn-state; type: dfn; text: [[state]];
+urlPrefix: https://w3c.github.io/geolocation-api/; spec: Geolocation
+    url: #watchposition-method; type: interface; for: Geolocation; text: watchPosition()
```

Comment:
These shouldn't be needed, as they are exported from that spec.

Reply:
Nothing is exported from ReSpec EDs since those all happen client side which the database can't pick up.

Reply:
The spec is on TR and evergreen, so should be picked up. If it's not, that's a Bikeshed-data/Shepherd issue (not a Respec issue).  

> Nothing is exported from ReSpec EDs since those all happen client side which the database can't pick up.

You are right, but that's like saying that a bikeshed ".bs" file is not picked up because the spec editor didn't run `bikeshed`. If you spot at spec that is not being snapshotted, please tell those folks to create a snapshot of it (exactly as one would for a .bs spec).

Reply:
TR can't be trusted to be evergreen, so as a matter of personal policy I don't encourage people to link to TR (and as a matter of WHATWG policy we never do from WHATWG specs).

Reply:
I think we should discuss these assumptions (elsewhere, sorry for taking us off topic): You are not wrong, but they are not entirely true either.

Reply:
Removed in fb99f17, thanks @marcoscaceres.

Reply:
:(